Cross.toml

```diff
@@ -0,0 +1,2 @@
+[target.aarch64-unknown-linux-gnu]
+image = "pinebuild"
```

Comment:
```suggestion
image = "pinephone"
```

Reply:
Thank you - it was late when I wrote that. Canged both to `pinephone-build` as that seems like a more descriptive name.